secrets.defaults.properties

```diff
@@ -0,0 +1,21 @@
+
+# Default values for the Mifos API
+# You can override these values by creating secrets.properties file in the root directory of the project
+
+# https://
+PROTOCOL_HTTPS = DEFAULT_VALUE
```

Comment:
Change all default values with current values present in the project like 

API_ENDPOINT = "/fineract-provider/api/v1"